styleguide/derek/view-templates/resource-center/resource-center.ejs

```diff
@@ -0,0 +1,93 @@
+        <div class="rsFilter-formTitle">Find a Resource</div>
+          <div class="form-item form-type-select form-item-tid-1">
+            <h6 class="rsFilter-fieldTitle">Solutions</h6>
+            <select class="rsFilter-formSelect form-select">
+              <option value="All" selected="selected">- Any -</option>
+              <option value="92">Public Cloud</option>
+              <option value="82">Private Cloud</option>
+            </select>
+          </div>
+          <h6 class="rsFilter-fieldTitle">Products</h6>
+          <div class="form-item form-type-select form-item-tid-2">
+            <select class="rsFilter-formSelect form-select">
+              <option value="All" selected="selected">- Any -</option>
+              <option value="101">AWS</option>
+              <option value="102">Azure</option>
+            </select>
+          </div>
+          <h6 class="rsFilter-fieldTitle">Resource Type</h6>
+          <div class="form-item form-type-check form-item-tid-3">
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="Webinar" value="Webinar">
+                <label class="rsFilter-formCheckLabel">Webinar</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="PDF" value="PDF">
+                <label class="rsFilter-formCheckLabel">PDF</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="Code" value="Code">
+                <label class="rsFilter-formCheckLabel">Code</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="Video" value="Video">
+                <label class="rsFilter-formCheckLabel">Video</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="Datasheet" value="Datasheet">
+                <label class="rsFilter-formCheckLabel">Datasheet</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="Support" value="Support">
+                <label class="rsFilter-formCheckLabel">Support</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="eBook" value="eBook">
+                <label class="rsFilter-formCheckLabel">eBook</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="Whitepaper" value="Whitepaper">
+                <label class="rsFilter-formCheckLabel">Whitepaper</label>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/5263371/25766662/652c60da-31b9-11e7-9e89-63c9186945ab.png)

On very narrow viewports the filter list wraps oddly.  Maybe the `sm` viewport should simulate what happens on `xs`?